checks/runchecks.go

```diff
@@ -110,3 +114,30 @@ func processTemplates(ctx *context.Context, r *pkg.CheckResult) *pkg.CheckResult
 
 	return r
 }
+
+func measureTestSeverity(ctx *context.Context, threshold *v1.TestThreshold) pkg.Severity {
+	if threshold == nil {
+		return pkg.SeverityInfo
+	}
+
+	thresholds := []struct {
+		severity pkg.Severity
+		expr     string
+	}{
+		{pkg.SeverityCritical, threshold.Critical},
```

Comment:
switch to duty constants

Reply:
Duty doesn't have those consts. created a PR for it https://github.com/flanksource/duty/pull/196